mesonbuild/dependencies/python.py

```diff
@@ -112,8 +113,13 @@ def sanity(self) -> bool:
         # Sanity check, we expect to have something that at least quacks in tune
 
         import importlib.resources
+        if sys.version_info >= (3, 13):
```

Comment:
Why not `>= (3, 9)`? Why use a deprecated function when the replacement is available?

Reply:
It didn't seem to matter, so just trying to make the minimal change here. If it did matter (as it has before with `importlib` and traversables, it's complex machinery), I'd rather find out through 3.13 early adopters.

It's also possible that something still changes before 3.13.rc1, and this would allow dropping the if-else.